include/mbedtls/hkdf.h

```diff
@@ -0,0 +1,112 @@
+/**
+ * \file hkdf.h
+ *
+ * \brief HKDF
```

Comment:
The `\brief` description isn't adequate and the Copyright header needs to be separated out from the `\brief` summary.